hashes/src/sha256t.rs

```diff
@@ -71,6 +71,7 @@ where
     pub fn from_engine(e: HashEngine) -> Hash<T> { from_engine(e) }
 
     /// Copies a byte slice into a hash object.
+    #[deprecated(since = "TBD", note = "Use `from_byte_array` instead.")]
     pub fn from_slice(sl: &[u8]) -> Result<Hash<T>, FromSliceError> {
```

Comment:
I would have expected this to call through to `from_byte_array` now like the other one.

Reply:
Done